src/views/Game/AIReview.tsx

```diff
@@ -883,6 +904,22 @@ export class AIReview extends React.Component<AIReviewProperties, AIReviewState>
             avg_score_loss[0] = btotal > 0 ? Math.round((10 * avg_score_loss[0]) / btotal) / 10 : 0;
             avg_score_loss[1] = wtotal > 0 ? Math.round((10 * avg_score_loss[1]) / wtotal) / 10 : 0;
 
+            score_loss_list[0].sort((a, b) => {
+                return a - b;
+            });
+            score_loss_list[1].sort((a, b) => {
+                return a - b;
+            });
+
+            median_score_loss[0] =
+                this.medianList(score_loss_list[0]) !== undefined
+                    ? Math.round(10 * this.medianList(score_loss_list[0])) / 10
+                    : 0;
+            median_score_loss[1] =
+                this.medianList(score_loss_list[1]) !== undefined
+                    ? Math.round(10 * this.medianList(score_loss_list[1])) / 10
```

Comment:
`round(x*10)/10` just truncates this number to the nearest 10th right?  You might be interested in the `toFixed()` function:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
I think at the moment it might be the case that the median returns undefined when the list is empty, so I'd still need some conditional operator in that table row. 
The alternative would be to return zero in that case I suppose. Defaulting to zero might make things easier I suppose.

Reply:
I've updated to use toFixed now for the mean, median and when calculating a percentage of the moves in a particular category.